pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -268,10 +300,23 @@ func (rc *reconciler) reconcile() {
 		if !rc.desiredStateOfWorld.VolumeExists(attachedVolume.VolumeName) &&
 			!rc.operationExecutor.IsOperationPending(attachedVolume.VolumeName, nestedpendingoperations.EmptyUniquePodName) {
 			if attachedVolume.GloballyMounted {
```

Comment:
who fills `attachedVolume.GloballyMounted` for block devices? IMO it's filled at the end of `attacher.MountDevice`, but that applies only to filesystem volumes, right?

Reply:
found it, `GenerateMapVolumeFunc`

Reply:
The attachedVolume.GloballyMounted is set via MarkDeviceAsMounted().
For filesystem volume case, MarkDeviceAsMounted() is called in the GenerateMountVolumeFunc().
And for Block volume case,  MarkDeviceAsMounted() is called in the GenerateMapVolumeFunc().

```
- GenerateMountVolumeFunc()
    -> volumeAttacher.MountDevice()
    -> MarkDeviceAsMounted()

- GenerateMapVolumeFunc()
     -> mapDevice()
     -> MarkDeviceAsMounted()
```